reference/deploy-tokens.html.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Deploy Tokens
+layout: docs
+sitemap: false
+nav: firecracker
+---
+
+<div class="border border-violet-600 bg-violet-50 rounded-l p-4 my-4 text-base text-navy">
+Deploy tokens are an experimental feature. They may be less reliable than traditional API tokens. Use them with caution and visit our <a href="https://community.fly.io">community forum</a> for help.
+</div>
+
+Whether you're [deploying your app with GitHub Actions](/docs/app-guides/continuous-deployment-with-github-actions/) or running your own CD service, it's best to avoid configuring deployment infrastrcutre with all-powerful tokens. Deploy tokens can be used with `flyctl` to manage a single application and it's resources.
+
+To get started, generated a deploy token on the Tokens tab of your app dashboard. Alternatively, run `flyctl auth token --deploy` to generate an app deploy token from the command line. Instruct `flyctl` to use your new token by setting it in the `FLY_API_TOKEN` environment variable.
```

Comment:
```suggestion
To get started, generate a deploy token on the Tokens tab of your app dashboard. Alternatively, run `flyctl auth token --deploy` to generate an app deploy token from the command line. Instruct `flyctl` to use your new token by setting it in the `FLY_API_TOKEN` environment variable.
```

This may be the first doc that positions the browser UI as the main way to do something and flyctl as an "alternative" 😄

Reply:
My first thought was "yes, but it's not too serious either way," but maybe people are in their web-based CI/CD thing already and the dashboard is more convenient. So I don't have an opinion. :) I'm happy for this to be merged.